.gitignore

```diff
@@ -74,3 +74,11 @@ tags
 .DS_Store
 .idea
 *.sublime-workspace
+
```

Comment:
Please remove these gitignore changes from the PR

Reply:
Isn't this just a newline at the end of the file?

Reply:
Hmm, you might be right. In which case the IDE settings will be forcing it